main/src/ca/uwaterloo/flix/language/phase/Lowering.scala

```diff
@@ -682,7 +682,7 @@ object Lowering extends Phase[Root, Root] {
   private def visitType(tpe0: Type)(implicit root: Root, flix: Flix): Type = {
     def visit(tpe: Type): Type = tpe match {
       case Type.KindedVar(id, kind, loc, rigidity, text) => kind match {
-        case Kind.Schema => Type.KindedVar(id, Kind.Star, loc, rigidity, text)
+        case Kind.SchemaRow => Type.KindedVar(id, Kind.Star, loc, rigidity, text)
```

Comment:
Disclaimer:
We only arrive here if we have a well-kinded schema row type variable not under a schema type constructor. This can only happen if it is a parameter to an enum e.g. `E[a: SchemaRow]`, which is not a useful construction (per previous discussion). So what we do here may be inconsequential.

Do you have an idea of what the right thing to do here is?

Reply:
I thought it should be `Schema`?

Reply:
I don't understand. There is no `Schema` kind (only `SchemaRow`).